.ci/setup.sh

```diff
@@ -22,6 +22,8 @@ then
             qpdf \
         || exit -1
 
-    Rscript -e "install.packages(c('assertthat', 'covr', 'data.table', 'futile.logger', 'httr', 'jsonlite', 'knitr', 'lintr', 'purrr', 'rmarkdown', 'stringr', 'testthat', 'uuid'), repos = 'https://cran.r-project.org')"
+    conda install -y -c conda-forge 'pandoc>1.12.3'
+
+    Rscript -e "install.packages(c('assertthat', 'covr', 'data.table', 'futile.logger', 'httr', 'jsonlite', 'knitr', 'lintr', 'purrr', 'rmarkdown', 'stringr', 'testthat', 'uuid'), repos = 'https://cran.r-project.org'), Ncpus = parallel::detectCores())"
```

Comment:
installing packages is taking 8+ minutes (!!!), so I'm taking the opportunity to also add `Ncpus = parallel::detectCores()` here in `install.packages()`.